HLTriggerOffline/Egamma/python/HLTmultiTrackValidatorGsfTracks_cff.py

```diff
@@ -0,0 +1,54 @@
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsEta.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPhi.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPhi.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPt.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPt.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXR.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXR.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXZ.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXZ.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsEta.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsEta.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPhi.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPhi.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPt.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPt.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXR.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXR.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXZ.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXZ.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.generalGpSelector.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.generalGpSelector.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.generalTpSelector.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.generalTpSelector.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.maxRapidityTP = cms.double( 3.0)
+hltGsfTrackValidator.minRapidityTP = cms.double(-3.0)
+hltGsfTrackValidator.maxEta = cms.double( 3.0)
+hltGsfTrackValidator.minEta = cms.double(-3.0)
+
+from Validation.RecoTrack.TrackValidation_cff import trackingParticlesElectron
+hltGsfTracksPreValidation = cms.Sequence(
+    cms.ignore(trackingParticlesElectron)    
+#    trackingParticlesElectron
+)
+
+hltMultiTrackValidationGsfTracks = cms.Sequence(
+    hltTPClusterProducer
+    + hltTrackAssociatorByHits
+#    + cms.ignore(trackingParticlesElectron)    
+    + hltGsfTracksPreValidation
```

Comment:
I don't understand why `hltGsfTracksPreValidation` is inserted in `hltMultiTrackValidationGsfTracks` sequence here, while they are inserted in `hltassociation` and `hltvalidation` sequences in `HLTValidation_cff.py`, respectively.

Following my comment there, I'd suggest to remove the `hltGsfTracksPreValidation` sequence altogether, uncomment the `trackingParticlesElectron` in above, and in `HLTValidation_cff.py` insert `hltMultiTrackValidationGsfTracks` in `hltassociation`.